### ISSUE TITLE
Missing Travis CI build status

### ISSUE THREAD

**Issue by sjakobi** [haskell-checkers/checkers#52]
The reporting of the Travis CI build status on PRs hasn't been working properly for a few months. See e.g. https://github.com/conal/checkers/pull/50#issuecomment-646553317, https://github.com/conal/checkers/pull/51#issuecomment-653876326.

One way to get this working again is to migrate from travis-ci.org to travis-ci.com. I have already done this for https://github.com/sjakobi/bsb-http-chunked.

I think the steps are:

1. On https://travis-ci.com/account/migrate subscribe to the "beta" at the bottom of the page
2. On the same page, select the `checkers` repo and click "Migrate selected repositories to travis-ci.com"

@conal Could you give this a try?

**Comment by conal** [comment]
@sjakobi Thanks for the instructions. I think I've migrated checkers to travis-ci.com now. Does checkers now need a travis.yaml file? How do Haskellers generate these files?

**Comment by sjakobi** [comment]
@conal Thanks! :)

Apparently the integration doesn't work quite yet: In https://github.com/conal/checkers/pull/53 there's no Travis status showing up.

Unfortunately I've done this process only once before for a repository of mine. There's some info at https://docs.travis-ci.com/user/migrate/open-source-repository-migration/.

At https://github.com/conal/checkers/settings/installations, is Travis listed as an installed app?

You could also try giving me admin permissions for this repo – maybe I can figure out the rest myself then.

**Comment by conal** [comment]
> At https://github.com/conal/checkers/settings/installations, is Travis listed as an installed app?

Yes.

> You could also try giving me admin permissions for this repo – maybe I can figure out the rest myself then.

Hm. I thought you already had full permissions as a collaborator. I'd be happy to grant more, but I don't know how.

**Comment by sjakobi** [comment]
> > You could also try giving me admin permissions for this repo – maybe I can figure out the rest myself then.
> 
> Hm. I thought you already had full permissions as a collaborator. I'd be happy to grant more, but I don't know how.

At https://github.com/conal/checkers/settings/access, you can try setting a "Role: Admin" for me.

**Comment by conal** [comment]
> At https://github.com/conal/checkers/settings/access, you can try setting a "Role: Admin" for me.

Hm. I don't any role options. Maybe because I have a free GitHub account?

**Comment by sjakobi** [comment]
Oh. It's possibly because this your personal account, not an organization. From https://docs.github.com/en/github/getting-started-with-github/access-permissions-on-github:

> While you can grant read/write access to collaborators on a personal repository, members of an organization can have more granular access permissions for the organization's repositories.

How about migrating this repo to an organization then, where I can have admin permissions? I'd call it `haskell-checkers` maybe?!

**Comment by conal** [comment]
Thanks for the suggestion. I'll make a new organization and migrate the repo.

**Comment by conal** [comment]
Done. Checkers is now under a new haskell-checkers org of which you're a member with full admin permissions.

**Comment by sjakobi** [comment]
Nice!

Via https://github.com/settings/applications I have requested access to this organization for Travis CI. I hope that I can manage the Travis settings once you've approved that.

**Comment by sjakobi** [comment]
The Travis integration finally seems to work! Thanks, @conal! :)

**Comment by conal** [comment]
Wonderful! Thanks for raising this issue and shepherding it through to resolution, @sjakobi!
